### ISSUE TITLE
highchart/highstock navigator doesn't work with keyboard

### ISSUE THREAD

**Issue by vakhsakhalyan** [highcharts/highcharts#14914]
I'm using hightchart navigator bar in bottom. I've included the accessibility js file to use all accessible components(keyboard navigation,screen reader). Now I'm trying to navigate to navigator bar using keyboard but it doesn't work. Are there any additional properties required?

# Description of the feature
highchart/highstock navigator doesn't work with keyboard

# Library related to the feature
Highcharts / Highcharts Stock, Accessibility

# Proof of Concept/Live example for the feature

![image](https://user-images.githubusercontent.com/22655825/104336869-4fb38400-550e-11eb-9c2c-6503f8c4f203.png)

----

**You can vote for this feature by adding a thumbs-up reaction to this post.**

**Comment by PaulDalek** [comment]
Hi @vakhsakhalyan

Thank you for contacting us and suggesting this enhancement.

FYI @oysteinmoseng

**Comment by oysteinmoseng** [comment]
We should consider if using the drag handles is the best way to provide this functionality for keyboard users, or if we want to deal with zoom/pan using shortcut keys instead - or both. Currently the only way is to use the Range Selector and input the date range.
